src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Bugzilla/Bugzilla52419.cs

```diff
@@ -1,37 +1,47 @@
-﻿#if ANDROID
-using NUnit.Framework;
+﻿using NUnit.Framework;
 using UITest.Appium;
 using UITest.Core;
 
 namespace Microsoft.Maui.TestCases.Tests.Issues;
 
 public class Bugzilla52419 : _IssuesUITest
 {
+#if ANDROID
+	const string TabPage1 = "TAB PAGE 1";
+	const string TabPage2 = "TAB PAGE 2";
+	const string BackButtonIdentifier = "";
+#else
+	const string TabPage1 = "Tab Page 1";
+	const string TabPage2 = "Tab Page 2";
+	const string BackButtonIdentifier = "Tab Page 1";
+#endif
+	const string PushNewPage = "PushNewPage";
+	const string AppearanceLabel = "AppearanceLabel";
+
 	public Bugzilla52419(TestDevice testDevice) : base(testDevice)
 	{
 	}
 
 	public override string Issue => "[A] OnAppearing called for previous pages in a tab's navigation when switching active tabs";
 
-	// [Test]
-	// [Category(UITestCategories.TabbedPage)]
-	// [FailsOnAndroidWhenRunningOnXamarinUITest]
-	// public void Bugzilla52419Test()
-	// {
-	// 	App.WaitForElement(q => q.Marked("Push new page"));
-	// 	App.Tap(q => q.Marked("Push new page"));
-	// 	App.WaitForElement(q => q.Marked("Push new page"));
-	// 	App.Tap(q => q.Marked("Push new page"));
-	// 	App.WaitForElement(q => q.Marked("Push new page"));
-	// 	App.Tap(q => q.Marked("Push new page"));
-	// 	App.Tap(q => q.Marked("Tab Page 2"));
-	// 	App.Tap(q => q.Marked("Tab Page 1"));
-	// 	App.Tap(q => q.Marked("Tab Page 2"));
-	// 	App.Tap(q => q.Marked("Tab Page 1"));
-	// 	App.Back();
-	// 	App.WaitForElement(q => q.Marked("AppearanceLabel"));
-	// 	var label = App.Query(q => q.Marked("AppearanceLabel"))[0];
-	// 	Assert.AreEqual("Times Appeared: 2", label.Text);
-	// }
+	 [Test]
+	 [Category(UITestCategories.TabbedPage)]
+	 public void Bugzilla52419Test()
```

Comment:
This test is failing on Android:
<img width="1140" alt="image" src="https://github.com/user-attachments/assets/6b294410-7616-45d0-8f29-a3c84fc7779e">

Reply:
Resolved CI failure and changes are committed.
<img width="388" alt="image" src="https://github.com/user-attachments/assets/a24978f1-d3cc-46a5-bf8a-7594552e0076">